logstash-core/spec/logstash/output_delegator_spec.rb

```diff
@@ -31,6 +31,7 @@
 
     context "after having received a batch of events" do
       before do
```

Comment:
Is a matter of style, but I do recommend using explicit params here, relaying in defaults it actually makes me think all time which was the default :+1: